src/Service/SerializerFactory.php

```diff
@@ -33,7 +35,7 @@ public static function createSerializer(
         JsonApiEncoder $jsonApiEncoder,
         EntityDenormalizer $entityDenormalizer,
         EntityNormalizer $entityNormalizer
-    ) {
+    ): SerializerInterface & NormalizerInterface {
```

Comment:
I don't get the usage of the composite type hint here. why not hint at  `Symfony\Component\Serializer\Serializer` as the return type of this method, since it's what actually gets returned? perhaps i'm missing something.

Reply:
Good call, I was being fancy because we don't need all the stuff on `Serializer`, but you're correct. Fixed.